plugins/out_stdout/stdout.c

```diff
@@ -222,9 +222,9 @@ static void cb_stdout_flush(struct flb_event_chunk *event_chunk,
             FLB_OUTPUT_RETURN(FLB_RETRY);
         }
 
-        while ((result = flb_log_event_decoder_next(
-                            &log_decoder,
-                            &log_event)) == FLB_EVENT_DECODER_SUCCESS) {
+        while ((flb_log_event_decoder_next(
+                                           &log_decoder,
```

Comment:
Either move `&log_decoder` to line 225 or decrease the indentation level.
Also remove the now unnecessary wrapping parenthesis.

Reply:
I modified them.